protected/humhub/components/ActiveRecord.php

```diff
@@ -280,4 +280,27 @@ public static function getObjectModel(): string
     {
         return static::class;
     }
+
+    /**
+     * Remove attributes from all scenarios
+     *
+     * @param array $scenarios
+     * @param string|array $attributes
+     * @since 1.18
+     */
+    public function removeScenarioAttributes(array &$scenarios, string|array $attributes): void
```

Comment:
@yurabakhtin Maybe `protected` and `static`?

Reply:
@luke- What reason to keep it as and `protected` and `static`?
If we make it `protected` then it is impossible to use it from outside the class like `ActiveRecord::removeScenarioAttributes()`.
If we don't need to use it outside then it is enough to make it only `protected`, otherwise `public` + `static`.
Am I wrong somewhere?

Reply:
@yurabakhtin I think, we should be restrictive as possible. So `protected` would be fine. Static because it doesn't use any instance properties. But we can leave it as it is.